mlflow/pipelines/steps/evaluate.py

```diff
@@ -284,7 +284,66 @@ def get_icon(validated):
                 + criteria_html,
             )
 
-        # Tab 2: SHAP plots.
+        # Tab 2: Classifier plots.
+        if self.template == "classification/v1":
+            classifiers_plot_tab = card.add_tab(
+                "Classifier Plots on the Validation Dataset",
+                "{{ CONFUSION_MATRIX }} {{CONFUSION_MATRIX_PLOT}}"
+                + "{{ LIFT_CURVE }} {{LIFT_CURVE_PLOT}}"
+                + "{{ PR_CURVE }} {{PR_CURVE_PLOT}}"
+                + "{{ ROC_CURVE }} {{ROC_CURVE_PLOT}}",
```

Comment:
What happens if none of the paths exists? Do we still render this tab?
Also can we add a quick test that this tab is rendered, similar to this: https://src.dev.databricks.com/mlflow/mlflow/-/blob/tests/pipelines/test_ingest_step.py?L559

Reply:
Per the [mlflow evaluate API](https://www.mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate), the plots will exist if classification. This tab is gated on classification template: https://github.com/prithvikannan/mlflow/blob/classifier-plots-clean/mlflow/pipelines/steps/evaluate.py#L309

Reply:
To test this, we would need to run the pipeline with `template: "classification/v1"`, which we cannot do until the classification PR is merged

Reply:
Makes sense for the test. :)
 
Also I agree that it is gated. But if we believe that the plots are going to exists. can we remove the if check for the path and just gate the entire tab with classification or not and assume that all plots would exists? What do you think? Or at least remove the `if` check from the plots that are required? So we never run into an issue where the code silently fails because of the `if` check rather than actually failing or something?

Reply:
@sunishsheth2009 good point. mlflow evaluate will always have a confusion matrix for classification, but lift curve plot, precision-recall plot, and ROC plot only exist for binary classification. for multi class classification there is a different set of plots. imo we should keep the `if` logic to allow for extension.